.github/workflows/ci.yml

```diff
@@ -0,0 +1,36 @@
+name: CI
+on:
+  push:
+    branches:
+      - master 
+  pull_request:
+jobs:
+  unit:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Scaladex
+        uses: actions/checkout@v2
+        with:
+          path: scaladex
+      - name: Checkout Scaladex small index
+        uses: actions/checkout@v2
+        with:
+          repository: scalacenter/scaladex-small-index
+          path: scaladex-small-index
+      - name: Checkout Scaladex contrib 
+        uses: actions/checkout@v2
+        with:
+          repository: scalacenter/scaladex-contrib
+          path: scaladex-contrib
+      - uses: olafurpg/setup-scala@v10
+        with:
+          java-version: "adopt@1.8"
+      - uses: coursier/cache-action@v5
+      - name: Run tests
+        run: |
+          cd scaladex
+          ./bin/scalafmt --test
+          sbt "data/run elastic"
+          sbt test
+          sbt server/universal:packageBin
```

Comment:
Could you make individual steps out of those?
So that it is easier to spot what is failing

Reply:
Sure!

Reply:
Changed to separate steps:
https://github.com/tgodzik/scaladex/runs/1380895668?check_suite_focus=true

what do you think?

Reply:
Perfect thank you!